src/fondue/Fondue.js

```diff
@@ -54,12 +54,9 @@ export default class Fondue {
 		const gposLangs = getLangs(this._font.opentype.tables.GPOS);
 		const allLangs = new Set([...gsubLangs, ...gposLangs]);
 
-		let langSys = {};
-		for (const lang of allLangs) {
-			if (languageMapping[lang]) {
-				langSys[lang] = languageMapping[lang];
-			}
-		}
+		const langSys = languageMapping.filter((l) =>
+			[...allLangs].includes(l.ot)
```

Comment:
This is now building an array (`[...allLangs]`) for every value in `languageMapping` for every element in `languageMapping`. Since `allLangs` is only used for lookups now, you could use `allLangs.has(l.ot)`, which is much more efficient.

Reply:
🤯